### PR TITLE
Adding missing Operative Exploits - mostly Armory

### PR DESCRIPTION
**Pull request by rmorgens**
Adding missing Operative Exploits, mostly from Armory

### CODE REVIEW

**Comment by rmorgens** [comment]
Addresses Issue#859 Trap Spotter Missing

File: src/items/class-features/concealed_weaponry_(ex).json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    "modifiers": [
+      {
+        "_id": "b055efaf-2f7c-4332-94e1-7d0802847beb",
+        "name": "Concealed Weaponry (Ex)",
+        "type": "insight",
+        "condition": "",
+        "effectType": "skill",
+        "enabled": true,
+        "max": 2,
+        "modifier": "round((@classes.operative.levels + 3) / 4) * 2",
```
Comment (LebombJames) [inline src/items/class-features/concealed_weaponry_(ex).json:95]: Use the updated operative's edge formula

File: src/items/class-features/concealed_weaponry_(ex).json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        "modifierType": "formula",
+        "notes": "bonus to Sleight of Hand checks when attempting to hide a small object on your body.",
```
Comment (LebombJames) [inline src/items/class-features/concealed_weaponry_(ex).json:97]: Include the "Double your operative's edge bonus..." here

File: src/items/class-features/disarming_attack_(ex).json [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      "value": null
+    },
+    "recharge": {
+      "charged": false,
+      "value": null
+    },
+    "requirements": "6th Level (Operative)",
+    "rollNotes": "",
+    "save": {
+      "type": "",
```
Comment (LebombJames) [inline src/items/class-features/disarming_attack_(ex).json:100]: Add the reflex save

File: src/items/class-features/knee_shot_(ex).json [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+      "value": null
+    },
+    "recharge": {
+      "charged": false,
+      "value": null
+    },
+    "requirements": "6th Level (Operative)",
+    "rollNotes": "",
+    "save": {
+      "type": "",
```
Comment (LebombJames) [inline src/items/class-features/knee_shot_(ex).json:100]: Add the reflex save

File: src/items/class-features/surveillance_wraith_(ex).json [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>You are invisible to technological surveillance systems and most forms of divination magic, per nondetection.</p>"
```
Comment (LebombJames) [inline src/items/class-features/surveillance_wraith_(ex).json:40]: Link to non-detection, and italicise it

File: src/items/class-features/trap_spotter_(ex).json [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+    "modifiers": [
+      {
+        "_id": "5c734048-5ae1-4b24-b072-3b0da4dcd927",
+        "name": "Trap Spotter (Ex)",
+        "type": "insight",
+        "condition": "",
+        "effectType": "skill",
+        "enabled": true,
+        "max": 2,
+        "modifier": "round((@classes.operative.levels + 3) / 4) * 2",
```
Comment (LebombJames) [inline src/items/class-features/trap_spotter_(ex).json:95]: Same notes as concealed weapon

**Review by LebombJames — CHANGES_REQUESTED** [review]
.

**Comment by LebombJames** [comment]
If you're updating details, remember to set these (and ones from other PRs) as class features

File: src/items/class-features/daring_explorer_(ex).json [hunk 7]
```diff
@@ -37,11 +37,49 @@
...
       "short": "",
       "unidentified": "",
-      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>When you fail an Acrobatics or Athletics check, you can spend 1 Resolve Point to either reroll the check and use the new result or add 5 to your result and use the new value to determine whether you succeed at the check. You can’t use this ability on @UUID[Compendium.sfrpg.class-features.MBNI94tbMoqizPUk]{Trick Attack (Ex)} attempts. Once you’ve used this ability, you can’t use it again until you’ve taken a 10-minute rest to recover Stamina Points.</p>"
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>When you fail an @Check[type:acrobatics] or @Check[type:athletics], you can spend 1 Resolve Point to either reroll the check and use the new result or add 5 to your result and use the new value to determine whether you succeed at the check. You can’t use this ability on @UUID[Compendium.sfrpg.class-features.MBNI94tbMoqizPUk]{Trick Attack (Ex)} attempts. Once you’ve used this ability, you can’t use it again until you’ve taken a 10-minute rest to recover Stamina Points.</p>"
```
Comment (LebombJames) [inline src/items/class-features/daring_explorer_(ex).json:40]: I recommend going over these again and ensuring `@Check` is only used when the ability calls for a check (from you or an enemy), not when it is simply talking about a skill/save/etc

File: src/items/class-features/elite_saboteur_(ex).json [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>You can attempt an @Check[type:engineering] to disable a device on a lock or trap in half the normal time. If this would reduce the duration to less than 1 round, you can disable the device as a standard action.</p>"
```
Comment (LebombJames) [inline src/items/class-features/elite_saboteur_(ex).json:40]: For things like this, you should add a Skill Check in the details tab so it appears on the chat card

File: src/items/class-features/feign_death_(ex).json [hunk 9]
```diff
@@ -30,21 +31,63 @@
...
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.On2ZrPJTe8FURsXY]{Operative Exploit}</p>\n<p>Whenever you take damage, you can attempt to feign death as a reaction. You immediately fall @UUID[Compendium.sfrpg.conditions.XeRGqHVtcZ7vsgJ0]{Prone} and attempt a @Check[type:bluff] against each opponent that is aware of you. The DC for this check is equal to 10 + your opponent’s total Sense Motive skill bonus, or 15 + 1-1/2 × the opponent’s CR, whichever is greater. Any opponent that you succeed against believes that you are dead and acts accordingly unless it inspects your body as a standard action and succeeds at a Medicine, Perception, or Sense Motive check at the same DC to uncover your deception. Maintaining this ruse is a full action each turn.</p>"
```
Comment (LebombJames) [inline src/items/class-features/feign_death_(ex).json:40]: This is another example of something that needs a Skill Check

**Review by LebombJames — CHANGES_REQUESTED** [review]
Just take another look over and ensure you're not including links/checks when they're not necessary (and to ensure you're using skill checks on the details tab too)

**Comment by rmorgens** [comment]
Replaced by PR#1412

**Pull request closed without merging**